### ISSUE TITLE
First request on nodejs / express app really slow

### ISSUE THREAD

**Issue by psegalen** [kflu/serman#9]
Hi,
I have a nodejs / express application hosted in serman, my first request on this app after the service has been started is always really long (about 40 seconds to 1 minute), it seems to occur also on the first request after 1 hour without any activity on the app.
I believe serman causes the problem because when I start the app locally in node, the first request takes under 3 seconds.
Does anyone know if I missed some piece of configuration or anything else?

**Comment by kflu** [comment]
Could you share your serman XML config?

It almost feels like your web app is running in an app pool on IIS or something that'll gets created / expired. That's why some requests take long to handle. I don't feel this is related to serman as it's just a wrapper to create a Window service.

**Comment by psegalen** [comment]
Here it is:
```
<service>
  <id>******</id>
  <name>******</name>
  <description>This service runs the ****** application</description>
  <executable>node.exe</executable>
  <arguments>"E:\Applications\******\index.js"</arguments>
  <logmode>rotate</logmode>
  <env name="NODE_ENV" value="production"/>
  <env name="PORT" value="8000" />
  <env name="SERVER_TYPE" value="serman"/>
</service>
```
No IIS is onvolved, it is supposed to host my Express app inside NodeJS, maybe Express could be the problem? But I doubt it since when I use it in a local console instance, it doesn't behave this way...

**Comment by kflu** [comment]
One thing you can check for sure is that when the **service** is running, check if your **node** process is present in task manager. If the node process is running, and your first request is still slow, that's a node/express problem. If the node process is not running, it's a serman/winsw problem. Then let's go from there.

**Comment by psegalen** [comment]
Okay, that's weird, I returned to work yesterday and sent you the XML config but I didn't test the behavior once again. When testing the presence of the node process in the task manager this morning, I saw that the behavior seems to be gone.
During my vacation, our sysadmin updated Windows Server 2016 with a huge patch (over 1 GB), we agreed that the problem was likely related to the OS stability and that the patch fixed it, I close this issue and will re-open it if the behavior appears once again.
Thank you for your time!
